text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/internal/TagPattern.java

```diff
@@ -0,0 +1,24 @@
+package net.kyori.adventure.text.minimessage.internal;
+
+import org.intellij.lang.annotations.Pattern;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.LOCAL_VARIABLE;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+
+@Documented
+@Pattern(TagInternals.TAG_NAME_REGEX)
+@Retention(RetentionPolicy.CLASS)
+@Target({ METHOD, FIELD, PARAMETER, LOCAL_VARIABLE }) //Unsure of what is needed here

```

Comment:
I'm not sure which of these, if any are needed or if I should just copy the ones the `@Pattern` annotation uses